src/app/app.module.ts

```diff
@@ -0,0 +1,27 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { ToastrModule } from 'ngx-toastr';
+import { EvenNumberPipe } from './even-number.pipe';
+import { HighlightDirective } from './highlight.directive';
+
+@NgModule({
+  declarations: [
+    AppComponent,
+    EvenNumberPipe,
+    HighlightDirective     //custom pipe
```

Comment:
That comment doesn' seem the correct

Reply:
Sorry, that was by mistake. Resolved now.